### PR TITLE
Update connect.me.xml ruleset

### PR DESCRIPTION
**Pull request by trevharmon**
The previous ruleset was pointing to another domain that was
generating a 404. It now simply replaces http with https. Also,
it now only handles www.connect.me.

### CODE REVIEW

File: src/chrome/content/rules/Connect.me.xml [hunk 1]
```diff
@@ -1,22 +1,16 @@
-<!--
-	^connect.me: Expired, missing certificate chain
-	www.connect.me: Refused
-
--->
 <ruleset name="Connect.me">
 
 	<!--	Complications:
 				-->
-  <target host="connect.me"/>
```
Comment (J0WI) [inline src/chrome/content/rules/Connect.me.xml]: Why did you remove this target?

File: src/chrome/content/rules/Connect.me.xml [hunk 2]
```diff
@@ -1,22 +1,16 @@
...
 	<!--	Redirect keeps path and args,
 		but not forward slash:
 					-->
-	<rule from="^http://(?:www\.)?connect\.me/+"
-		to="https://www.respectnetwork.com/connect-me-2/" />
+	<rule from="^http:"
+		to="https:" />
 
-		<test url="http://connect.me//" />
+		<test url="http://www.connect.me/" />
```
Comment (J0WI) [inline src/chrome/content/rules/Connect.me.xml]: All listed targets are implicit test urls.

File: src/chrome/content/rules/Connect.me.xml (hunk 1 above)
Comment (trevharmon) [inline src/chrome/content/rules/Connect.me.xml]: `connect.me` and `www.connect.me` both point to different hosts. At the moment, `connect.me` is undergoing some changes, so the https redirects weren't happening correctly. My intention was to get `www.connect.me` fixed properly and then, when `connect.me` became stable to submit another PR to update that URL.

File: src/chrome/content/rules/Connect.me.xml (hunk 2 above)
Comment (trevharmon) [inline src/chrome/content/rules/Connect.me.xml]: Should they just removed then? At the moment, we aren't needing any wildcards on this domain at this time.

File: src/chrome/content/rules/Connect.me.xml (hunk 1 above)
Comment (J0WI) [inline src/chrome/content/rules/Connect.me.xml]: We use to add comments for non-working hosts. Please document that `connect.me` serves an incomplete certificate chain.

File: src/chrome/content/rules/Connect.me.xml (hunk 2 above)
Comment (J0WI) [inline src/chrome/content/rules/Connect.me.xml]: I would merge it anyway, but since I requested other changes you can also remove this redundant test url.

**Comment by trevharmon** [comment]
Made requested changes to the ruleset definition.

**Comment by J0WI** [comment]
Thanks!
